homoalg.tex

```diff
@@ -2018,7 +2018,7 @@ \section{Komplexe und (Ko-)Homologie}
 
 \begin{defn}
   Sei $\mathcal{B}$ ein kohomologisches Koeffizientensystem auf einer simplizialen Menge $X$. Wir setzen
-  \[ C^n(X; \mathcal{B}) \coloneqq \{ \text{ Funktionen } f : (\sigma \in X_n) \to \mathcal{B}_\sigma \, \} \]
+  \[ C^n(X; \mathcal{B}) \coloneqq \{ \text{ Funktionen } f : X_n \to \coprod_{\sigma \in X_n} \mathcal{B}_\sigma | f(\sigma) \in X_n \, \} \]
```

Comment:
Da habe ich dich wohl mit meiner Notation verwirrt! Ich habe hier etwas Notation von abhängiger Typtheorie geborgt. Eine abhängige Funktion ist eine Funktion, bei der die Zielmenge nicht statisch ist, sondern von der Eingabe abhängen kann. In diesem Fall ist die Eingabe das sigma aus X_n. Ein anderes Beispiel: Ein Vektorfeld auf einer Mannigfaltigkeit M ist eine Abbildung `X : (p \in M) --> T_p M`. Die von Mathematikern meistbenutzte Notation für solche abhängigen Funktionen ist wohl `f = (f_sigma \in B_sigma)_{sigma \in X_n}`.